src/API/JobProcessorBase.cs

```diff
@@ -88,7 +88,10 @@ protected async Task SubmitPipelineJob(string jobName, string pipelineId, JobPri
 
             _logger.Log(LogLevel.Information, "Queueing a new job '{0}' with pipeline '{1}', priority={2}, instance count={3}", jobName, pipelineId, jobPriority, instances.Count);
 
-            var job = await _jobsApi.Create(pipelineId, jobName, jobPriority);
+            var metadata = new JobMetadataBuilder();
+            metadata.AddSource($"{Name}-{AeTitle}");
```

Comment:
Perhaps better to add these as separate details? The AET makes sense, not sure if the DA name for it is relevant, but if it is shouldn't this be `AddSourceName`?

Alternatively, wouldn't a function `AddCustomTag` be more general and widely applicable? e.g. `AddCustomTag('Source', 'myAET')` `AddCustomTag('SourceName', 'my-aet')`?

Reply:
I didn't use `AddCustomTag` because the class itself is inherited from Dictionary and the main reason is to hide the "key" from the user.